tests/Model/LazyLoading/ManyToManyRelationTest.php

```diff
@@ -342,15 +342,33 @@ public function testLocalizedBrickAttributes()
         //prepare data object
         $object = $this->createDataObject();
         $brick = new LazyLoadingLocalizedTest($object);
-        $brick->getLocalizedfields()->setLocalizedValue('lrelations', $this->loadRelations()->load());
+        $brick->getLocalizedfields()->setLocalizedValue('lrelations', $this->loadRelations()->load(), 'en');
+        $brick->getLocalizedfields()->setLocalizedValue('lrelations', $this->loadRelations()->load(), 'de');
 
         $object->getBricks()->setLazyLoadingLocalizedTest($brick);
         $object->save();
 
         $brick->setLInput(uniqid());
         $object->save();
         $object = Concrete::getById($object->getId(), true);
-        $this->assertTrue(count($object->getBricks()->getLazyLoadingLocalizedTest()->getLRelations()) > 0);
+        $this->assertTrue(count($object->getBricks()->getLazyLoadingLocalizedTest()->getLRelations('en')) > 0);
```

Comment:
test for other relation types?..at-least for many-to-one?

Reply:
@dvesh3 added a test for many to one